test/common/upstream/load_balancer_simulation_test.cc

```diff
@@ -32,6 +32,54 @@ static HostSharedPtr newTestHost(Upstream::ClusterInfoConstSharedPtr cluster,
                    envoy::api::v2::endpoint::Endpoint::HealthCheckConfig::default_instance())};
 }
 
+// Simulate weighted LR load balancer.
+TEST(DISABLED_LeastRequestLoadBalancerWeightTest, Weight) {
```

Comment:
Usually I prefix the actualy test with `DISABLED_`, i.e. `DISABLED_weight`, but I guess this also works? TIL.